app/packs/src/components/reactQuill/ToolbarDropdown.js

```diff
@@ -30,14 +30,6 @@ export default class ToolbarDropdown extends React.Component {
   }
 
   componentDidMount() {
-    const cssRule = ".ql-snow .ql-picker." + this.ddQuillId + " .ql-picker-label::before, ." + this.ddQuillId + " .ql-picker.ql-size .ql-picker-item::before { content: attr(data-label); line-height: 25px; }";
```

Comment:
🥳 

Can we now get rid of `this.ddQuillId` as well?

Reply:
Yes, I' ll try to remove it